frontend/app/src/pages/import/index.vue

```diff
@@ -3,26 +3,31 @@ const { t } = useI18n();
 </script>
 
 <template>
-  <VContainer>
-    <CardTitle class="mt-2">{{ t('import_data.title') }}</CardTitle>
-    <div class="text-rui-text-secondary">
-      {{ t('import_data.description') }}
-    </div>
-    <RuiAlert
-      outlined
-      type="warning"
-      class="mt-4"
-      :title="t('import_data.notice_warning')"
-    >
-      <i18n tag="span" path="import_data.notice">
-        <template #link>
-          <ExternalLink url="https://github.com/rotki/rotki/issues/new/choose">
-            {{ t('import_data.notice_link') }}
-          </ExternalLink>
-        </template>
-      </i18n>
-    </RuiAlert>
+  <div class="p-4">
+    <TablePageLayout :title="[t('import_data.title')]">
+      <template #buttons>
+        <HintMenuIcon>
+          {{ t('import_data.description') }}
+        </HintMenuIcon>
```

Comment:
<img width="1164" alt="image" src="https://github.com/rotki/rotki/assets/26648140/fbb49762-9d01-428f-a788-9ebfb905877e">

The menu is too big, maybe you can set max-width there?

Reply:
Let me do some changes there